README.md

```diff
@@ -416,321 +412,20 @@ Python | 3.6 or higher | No | *N/A* | *Pre-installed or Installed by user* | [PS
-
-
-### Build Options
-When building oneMKL the SYCL implementation can be determined, by setting the `ONEMKL_SYCL_IMPLEMENTATION` option. Possible values are 
-- `dpc++` (default) for the [Intel(R) oneAPI DPC++ Compiler](https://software.intel.com/en-us/oneapi/dpc-compiler) and for the `clang++` from [Intel project for LLVM* technology](https://github.com/intel/llvm/releases) compilers. 
-- `hipsycl` for the [hipSYCL](https://github.com/illuhad/hipSYCL) SYCL implementation
-
-All options specified in the Conan section are available to CMake. You can specify these options using `-D<cmake_option>=<value>`.
-
-The following table provides a detailed mapping of options between Conan and CMake.
-
-Conan Option | CMake Option | Supported Values | Default Value 
- :---------- | :----------- | :--------------- | :---          
-build_shared_libs        | BUILD_SHARED_LIBS        | True, False         | True      
-enable_mklcpu_backend    | ENABLE_MKLCPU_BACKEND    | True, False         | True      
-enable_mklgpu_backend    | ENABLE_MKLGPU_BACKEND    | True, False         | True      
-*Not Supported*          | ENABLE_CUBLAS_BACKEND    | True, False         | False     
-*Not Supported*          | ENABLE_CUSOLVER_BACKEND  | True, False         | False     
-*Not Supported*          | ENABLE_CURAND_BACKEND    | True, False         | False     
-*Not Supported*          | ENABLE_NETLIB_BACKEND    | True, False         | False     
-*Not Supported*          | ENABLE_ROCBLAS_BACKEND   | True, False         | False     
-enable_mklcpu_thread_tbb | ENABLE_MKLCPU_THREAD_TBB | True, False         | True      
-build_functional_tests   | BUILD_FUNCTIONAL_TESTS   | True, False         | True      
-build_doc                | BUILD_DOC                | True, False         | False     
-target_domains (list)    | TARGET_DOMAINS (list)    | blas, lapack, rng   | All domains 
-
-*Note: `build_functional_tests` and related CMake option affects all domains at a global scope.*
-
-*Note: When building with hipSYCL, `-DHIPSYCL_TARGETS` additionally needs to be provided according to the targeted hardware. For the options, see the tables in the hipSYCL specific sections*
-
----
-
-## Project Cleanup
-
-Most use-cases involve building the project without the need to cleanup the build directory. However, if you wish to cleanup the build directory, you can delete the `build` folder and create a new one. If you wish to cleanup the build files but retain the build configuration, following commands will help you do so. They apply to both `Conan` and `CMake` methods of building this project.
-
-```sh
-# If you use "GNU/Unix Makefiles" for building,
-make clean
-
-# If you use "Ninja" for building
-ninja -t clean
-```
+## Documentation
+- [Contents](https://oneapi-src.github.io/oneMKL/)
+- [About](https://oneapi-src.github.io/oneMKL/introduction.html)
+- [Get Started](https://oneapi-src.github.io/oneMKL/selecting_a_compiler.html)
```

Comment:
Same "File not found" error.

Reply:
(New HTML not published to GitHub Pages yet.)

Reply:
Could you explain how the order works if possible? I mean, say we merged this PR. When/how do these new rst files are generated so that there are no issues in README links between the multiple steps?

Reply:
Good question. When we merge the PR, we should go ahead and publish right away. (I was mistaken in suggesting that it might not happen until the release.) Fortunately, publishing to GitHub Pages is quick. Someone (typically Maria) just has to merge a special PR to a dedicated GitHub Pages branch. She could wait to merge the present PR until merging the GitHub Pages one.

Reply:
Thanks for the explanation. It makes sense now. Agreed, also doing the two merges together prevents any potential mismatches. 
I don't think, there is much I can check in this PR without the other files. :) I say, let's merge both and then do a final check.